tests.live/Python/Operations.qs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/// Q# code should be in one or more .qs files that live 
+/// in the same directory as the python classical driver.
+///
+
+namespace Microsoft.Quantum.SanityTests {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+
+    /// # Summary
+    /// A quantum random number generator with a variable number of qubits.
+    operation QRNG(count : Int, name : String) : Result[] {
+        Message($"Hello {name} again!");
+
+        mutable r = new Result[count];
```

Comment:
I think the `new 'T[count]` syntax will soon be deprecated (https://github.com/microsoft/qsharp-compiler/pull/956), such that it may be worth using the new syntax early:
```suggestion
        mutable r = [Zero, size = count];
```